packages/docsearch-react/src/DocSearchModal.tsx

```diff
@@ -27,6 +27,13 @@ export interface DocSearchModalProps extends DocSearchProps {
   onClose?(): void;
 }
 
+const setRealHeight = () => {
+  // We update the value of the CSS variables
```

Comment:
```suggestion
  // We rely on a CSS property to set the modal height to the full viewport height
  // because all mobile browsers don't compute their height the same way.
  // See https://css-tricks.com/the-trick-to-viewport-units-on-mobile/
```

We should explain _why_ in the comments, not _what_.